active_model_serializers.gemspec

```diff
@@ -47,4 +47,5 @@ Gem::Specification.new do |spec|
 
   spec.add_development_dependency 'bundler', '~> 1.6'
   spec.add_development_dependency 'timecop', '~> 0.7'
+  spec.add_development_dependency 'grape', '~> 0.13.0'
```

Comment:
Just curious, do you mean to be `0.13.x` where `x >= 0` or `0.x` where `x >= 13`?. If the latter, it should be `~> 0.13`